test-data/phony-guests/win7-32-software.reg

```diff
@@ -0,0 +1,10 @@
+
+
```

Comment:
I think this is concatenated after the generic registry entries, so we need a blank line, but probably not two blank lines?